### ISSUE TITLE
HTML5 export Non UTF-8 encoded characters for upload on Itch.io

### ISSUE THREAD

**Issue by Bouh** [4ian/GDevelop#3679]
## Describe the bug

Export a game with assets containing accents from an HTML 5 export to Itch.io return an error on Itch: 
![image](https://user-images.githubusercontent.com/1670670/154931151-3ada0bdb-d2d1-441f-b251-087cead9dfee.png)

This is a pain point that users have encountered during the process of submitting a game to the jam.

This happened when a file in the project have a accent like `é`

## Other details
* All versions

**Comment by 4ian** [comment]
Yep we already rename resources at export in case of filename conflict, we should do the same to ensure filenames only have ASCII characters (i.e: if a resource is called "ArrièrePlanÉté.png", it should probably be renamed to "Arri_xx_rePlan_xx_t_xx_" with xx being some numbers to designate the unicode code point OR renamed to "ArrirePlant-1.png", i.e: we remove all non ASCII characters and add a unique number to avoid conflicts).

**Comment by Oxey405** [comment]
Hello ! Can I try to work on this for a bit ?

**Comment by Bouh** [comment]
Sure feel free to give it a try when you see no one working already on it, no need for asking before ;)
